src/react/LoginPage.jsx

```diff
@@ -16,19 +16,13 @@
    limitations under the License.
 */
 
-import React, { useState } from "react";
-import { Link } from "react-router-dom";
-import {
-  Button,
-  Col,
-  Container,
-  Form,
-  Row
-} from "react-bootstrap";
-import { isValidRealmName, validateAstarteToken } from "./AstarteClient.js";
+import React, { useState } from 'react';
+import { Button, Col, Container, Form, Row } from 'react-bootstrap';
+import { isValidRealmName, validateAstarteToken } from './AstarteClient';
 
 function isValidUrl(urlString) {
   try {
+    // eslint-disable-next-line no-new
     new URL(urlString);
```

Comment:
this can probably be replaced with `URL(urlString)`. We only need to see if the creation throws an error

Reply:
Unfortunately that is not possible and would result in:

`Uncaught TypeError: Failed to construct 'URL': Please use the 'new' operator, this DOM object constructor cannot be called as a function`